app/english/vocabulary/page.tsx

```
@@ -3,5 +1,16 @@
+import { FadeIn } from "@/components/animations/fade-in";
+import { VocabularyPractice } from "@/components/English/vocabulary-practice";
+import { createClient } from "@/lib/supabase/server";
+import next from "next";
+import { cookies } from "next/headers";
+
+export default async function VocabularyPage() {
+  const cookieStore = cookies();
+  const supabase = createClient(cookieStore);
```

ESLint

[error] 'supabase' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)